ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/CCodeGeneratorAmazonS3InitRequests_Test.java

```diff
@@ -158,7 +172,10 @@ public void testGenerateInitAmazonS3WithOptionalQueryParams() throws IOException
                 ImmutableList.of(
                         new Ds3Param("Job", "java.util.UUID", true),
                         new Ds3Param("Offset", "long", false)),
-                ImmutableList.of()));
+                ImmutableList.of()),
+                new Ds3DocSpecImpl(
```

Comment:
Can use `Ds3DocSpecEmptyImpl` for an empty doc spec instead of initializing with empty maps.